### ISSUE TITLE
Repostitory structure for master is not compliant

### ISSUE THREAD

**Issue by RoldyBuffalo** [korylprince/hassio-apcupsd#21]
This is the error I get when trying to install a fresh copy in hassio.

I'm following the directions as best as I know how, what could I be doing wrong here?

**Comment by korylprince** [comment]
Not sure what went wrong on your end. On a new a new HassOS install I:

* Turned on Advanced Mode in user profile
* Go to Supervisor tab -> Add-on Store -> Three-dot menu -> Repositories
* Add the "https://github.com/korylprince/hassio-apcupsd" repository
* Install one of the two apcupsd add-ons
* Start the add-on

All of this worked like it should. Are you sure you're up to date?

**Comment by RoldyBuffalo** [comment]
I tell you what, I'm not sure where I missed it, but I guess I didn't know it was a supervisor addon, I was adding it to hacs!

Crumbs for anyone trying to add this to hacs, it doesn't work, for obvious reasons, maybe too obvious. It's a repo, I should have known. I got it all squared away with my Cyber Power UPS 1500, it's even a pretty old model, so I'm happy. 

Once I added it to supervisor, instead of hacs, it worked as intended no issues. Leave it to me to beans something up this easy.

![image](https://user-images.githubusercontent.com/58420778/85209142-11990d80-b2fb-11ea-9f00-75789af4a0bf.png)
